App/NativeModules/Textile/__tests__/Api.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    acceptExternalThreadInvite,
+    acceptThreadInviteViaNotification,
+    addExternalThreadInvite,
+    addSchema,
+    addThread,
+    addThreadFiles
+} from '../Api'
+import { IDirectory } from '../pb/textile-go'
+
+const threadId = 'QmdNgTtH468cqZFzXCi4sVSWTbJMWQbhYb8cBVyikP9LzW'
+const threadKey = 'VsHHHz8bC8fu9k78RaX8ujQsUzGzaUxwKJyLFKKDacUZoWJaouGnzUQwgmh5'
+const threadName = 'Great Name'
+
+describe('textile api', () => {
```

Comment:
happy to remove these tests completely if it doesn't seem valuable. ended up not tackling sagas today....

Reply:
Lots of good stuff in here, I think it's valuable for sure. We can keep expanding this test to cover all the functions?

Reply:
for sure